### PR TITLE
Show OOM infos in post fail hook

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/65040
Verification: http://kazhua.qa.suse.de/tests/94#step/oom/25

### CODE REVIEW

File: lib/utils.pm [hunk 1]
```diff
@@ -1402,6 +1403,25 @@ sub show_tasks_in_blocked_state {
...
+
+ show_oom_info
+
+Show logs about an out of memory process kill.
+
+=cut
+sub show_oom_info {
+    if (script_run('dmesg | grep "Out of memory"') == 0) {
+        my $oom = script_output('dmesg | grep "Out of memory"');
+        if (!check_var('BACKEND', 'svirt')) {
```
Comment (okurz) [inline lib/utils.pm:1416]: can you use `has_ttys` from lib/Utils/Backends.pm?

**Review by okurz — COMMENTED** [review]
very nice :) Just one question in line

File: lib/utils.pm (hunk 1 above)
Comment (asdil12) [reply lib/utils.pm]: done

**Review by foursixnine — APPROVED** [review]

**Comment by mdoucha** [comment]
This change is breaking LTP tests which trigger OOM intentionally.

**Comment by asdil12** [comment]
@mdoucha can you link an example?

**Comment by mdoucha** [comment]
Both tests are expected to trigger OOM killer:
https://openqa.suse.de/tests/4659049#step/fork12/12
https://openqa.suse.de/tests/4659050#step/min_free_kbytes/12

**Pull request merged**